packages/example/src/services/metamask.ts

```diff
@@ -26,11 +26,15 @@ export const pluginOrigin = `wallet_plugin_${origin}`;
 
 export async function installPolkadotSnap(): Promise<boolean> {
     try {
-        await web3Enable('my cool dapp');
-        console.log("Snap installed!!");
+        await window.ethereum.send({
```

Comment:
@mpetrunic @irubido I am not sure if I am missing some information as I wasn't involved in this part of code as much, but the problem was that snap was never enabled and permissions requested. This works now but I am not sure if it is ok to use this code snip for enabling snap.

Reply:
nope, not ok, probably means enable function inside adapter isn't implemented properly

Reply:
@mpetrunic I see now, I will try few more things to solve this bug :)